internal/adminapi/backoff_strategy_konnect.go

```diff
@@ -158,7 +159,7 @@ func (s *KonnectBackoffStrategy) whyCannotUpdate(
 	if isTheSameFaultyConfig {
 		reasons = append(reasons, fmt.Sprintf(
 			"config has to be changed: %q hash has already failed to be pushed with a client error",
-			string(s.lastFailedConfigHash),
+			base64.StdEncoding.EncodeToString(s.lastFailedConfigHash),
```

Comment:
Why base64 encode the hash 🤔?

Reply:
Oh ok, it's a `[]byte` 👍

Reply:
Yeah, I agree that's gonna be better approach 👍 Thanks!